README.md

```diff
@@ -91,6 +93,8 @@ rm -rf ~/.meteor/
 sudo rm /usr/local/bin/meteor
 ```
 
+On Windows, just run the uninstaller.
```

Comment:
Can you say a little more here about going to "Control Panel" > "Uninstall a program"?